src/back/models/attributes/Attribute.js

```diff
@@ -222,6 +225,11 @@ function Attribute() {
       'Invalid property "name" when creating an Attribute (it has to be a ' +
       'string)'
     );
+    expect(_reservedNames).to.not.include(
+      attribute.name,
+      'Invalid property "name" when creating an Attribute (it must not ' +
```

Comment:
Should check for reserved name on the else clause too (around line 299), since the Attribute constructor accepts two argument forms: object or list of properties.